fillin/src/main/java/com/teamfillin/fillin/domain/account/Account.java

```diff
@@ -0,0 +1,34 @@
+package com.teamfillin.fillin.domain.account;
+
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
```

Comment:
Table 명 명시하면 좋을것같아요!
추가로, 복합 unique key 도 해당 영역에서 설정가능합니다.

```suggestion
@Entity
@Table(
	name = "account",
	indexes = {
		@Index(name = "idx_unique_01", columnList = "socialType, socialId")
	}
)
```

<img width="801" alt="스크린샷 2023-11-19 오후 2 05 30" src="https://github.com/TeamFILL-IN/server-renew/assets/88091704/b343281a-ccfb-47e6-a413-07715b34fbcc">



- 원래 기존에 unique Key 에 user_no 도 넣었는데, 제거해도 될것같습니다!

Reply:
오.. 이런식으로 하는게 가능하군요!! 

> indexes = {
> 		@Index(name = "idx_unique_01", columnList = "socialType, socialId")
> 	}

이 부분이 복합 unique key 를 설정하는 코드인가요?

Reply:
정확히는 unique index 는 db 에서 설정하는건데요, (위처럼 안하고, db 구성시 직접 쿼리로 세팅)
위처럼 해두면 명시적으로 index 가 어떻게 구성되어있는지 알기 쉽고,
jpa auto ddl 에 적용됩니다